lib/registries/addon/components/registries-navbar/component.ts

```diff
@@ -54,7 +54,7 @@ export default class RegistriesNavbar extends AuthBase {
         if (this.provider.id === 'egap') {
```

Comment:
@brianjgeiger, with these changes, is there any need to keep this `egap` special case?

Reply:
No, egap is now waffled on for everyone in production anyways, so this is ready to go away whenever.